content/blog/2016-05-09-when-to-use-varbit-chunks.md

```diff
@@ -0,0 +1,159 @@
+---
+title: When (not) to use varbit chunks
+created_at: 2016-05-09
```

Comment:
We might not want to time it for the same date as the other planned blog post tomorrow.